### DISCUSSION TITLE
The Release Package size is different in Github releases and Pypi.

### DISCUSSION THREAD

**Discussion by rolypolybunz** [python-restx/flask-restx#627] [Q&A]
Hi Team,

I have query on release package size differences in Github vs hosted official Pypi repo. The difference is also quite a lot (400 Kb in github vs ~2MB in Pypi). I got to understand this when my swagger UI didn't come up due to missing static directory containing the required css files. Any specific reason to the reduction in the size here?

Thanks!
Anand

**Comment by peter-doggart** [comment]
As you point out, the PyPi released sources and wheels contain the Swagger UI components, which don't live in this repo. If you clone this repo you can run `inv assets` which collects the required components. 

Would it be useful if the GitHub releases also contained these? Currently, the releases are generated by the tag but not the build actions, so it just doesn't know to include it.

**Reply by rolypolybunz** [reply]
Hi Peter,

Thanks for responding back! usually with Github releases, its always a fully built sources that are provided. Unless any issue with licensing, it would be very ideal that the release builds are identical in Github releases and in the Official public Pypi repo. Also, a small note mentioning the difference in the release package would also be very helpful(I couldn't find any reference, at least not highlighted).

Thanks!
Anand
